src/Container.php

```diff
@@ -121,6 +121,15 @@ public function reset(): void
     }
 
     /**
+     * Fast checking if the all services are available and dependencies can be resolved gracefully
+     * Useful for unit tests.
+     */
+    public function validate(): void
+    {
+        array_map([$this, 'get'], array_keys($this->definitions));
```

Comment:
maybe make checking all of the definitions and accumulate errors?

Reply:
This might be useful